Source/Flow/Private/Nodes/FlowNodeBase.cpp

```diff
@@ -633,12 +628,7 @@ void UFlowNodeBase::LogWarning(FString Message)
 #endif
 }
 
-void UFlowNodeBase::LogWarningConst(FString Message) const
-{
-	const_cast<UFlowNodeBase*>(this)->LogWarning(Message);
-}
-
-void UFlowNodeBase::LogNote(FString Message)
+void UFlowNodeBase::LogNote(FString Message) const
```

Comment:
I would keep separate const and const-version as it sometimes nodes need to modify some internal properties